packages/stats/gbstats/models/statistics.py

```diff
@@ -194,9 +199,93 @@ def create_joint_statistic(
     )
 
 
+@dataclass
+class QuantileStatistic(Statistic):
+    n: int  # number of events here
+    nu: float
+    q_hat: float  # sample estimate
+    q_lower: float
+    q_upper: float
+    main_sum: float  # numerator sum
+    main_sum_squares: float
+    denominator_sum: float  # denominator sum
+    denominator_sum_squares: float
+    main_denominator_sum_product: float
+
+    # Don't we need to similarly handle cases where the sample size and quantile range would make us go < 0 or > 1?
+    # We can return the quantile itself, but can't do inference so in those cases we need to output _default_output() as well.
+
+    @property
+    def _has_zero_variance(self) -> bool:
+        return self.variance_init <= 0.0 and self.n < 100
+
+    @property
+    def mean(self) -> float:
+        return self.q_hat
+
+    @property
+    def unadjusted_mean(self) -> float:
+        return self.mean
+
+    @property
+    def variance_init(self) -> float:
+        multiplier = scipy.stats.norm.ppf(0.975, loc=0, scale=1)
```

Comment:
Doesn't 0.975 need to be computed from some `alpha` from the configuration? Maybe `QuantileStatistic` also needs to take `alpha` as an input and use it here and use it when computing the `n` for the issue of requesting a quantile < 0 or > 1 (the other comment above?)

Reply:
I thought we agreed that we would always use .975 when estimating uncertainty.  Happy to chat more about this.

Reply:
updated to alpha